assets/src/blocks/Happypoint/useHubspotEmbedCode.js

```diff
@@ -1,42 +1,47 @@
+/* eslint-disable jsdoc/require-param-type */
+/* eslint-disable no-undef */
 import { useScript, removeScript } from '../../components/useScript/useScript';
 
 /**
  * Find Hubspot parameters in the embed code
  * Load the Hubspot form if found
+ *
+ * @param  root0
```

Comment:
I'm not sure what these `root0` are 🤔 Tbh I think that we can just remove this doc, it also still shows linting errors:
<img width="474" alt="Screenshot 2022-12-16 at 11 28 15" src="https://user-images.githubusercontent.com/6949075/208078595-fad6dc3e-f11f-4748-88bc-2b8738193855.png">

Reply:
True, done.